### ISSUE TITLE
L1GtUtils errors

### ISSUE THREAD

**Issue by mulhearn** [cms-l1t-offline/cmssw#47]
DL writes:

Amongst the other messages that have come into 76x, the loudest ones are

%MSG-e HLTConfigData: PATTriggerProducer:patTrigger 17-Sep-2015 23:28:17 CEST Run: 1 Event: 2001 Error in determining HLT prescale set index from L1 data using L1GtUtils: Tech/Phys error = 210110/210110 Tech/Phys psfsi = -1/-1 %MSG %MSG-e HLTConfigData: PATTriggerProducer:patTrigger 17-Sep-2015 23:28:17 CEST Run: 1 Event: 2001 Error in determining L1T prescales for HLT path: 'HLT_PFJet200_v2' with complex L1T seed: 'L1_SingleJet128' using L1GtUtils: isValid=1 l1tname/error/prescale 1 0:L1_SingleJet128/210110/-1.
%MSG

VG writes::

Basically, the error code says:

200000 - Retrieve L1 trigger configuration from event setup only.
210000     Event setup valid.

but

L1 configuration valid
Error: L1GlobalTriggerReadoutRecord with requested InputTag not found in
the event

L1 configuration valid
Error: L1GlobalTriggerRecord with requested InputTag not found in the event

As the prescale factor index is retrieved in this use case  from
L1GlobalTriggerReadoutRecord or L1GlobalTriggerRecord, if none exists
with the requested input tag, L1GtUtils will return an error code.

The first thing which must be done is to look at the event content.

**Comment by mulhearn** [comment]
Dated issue.  Closing.
